src/rgw/rgw_kafka.cc

```diff
@@ -632,6 +633,11 @@ class Manager {
       ldout(cct, 1) << "Kafka connect: user/password are only allowed over secure connection" << dendl;
       return false;
     }
+    
+    if (brokers.has_value()) {
+      broker = brokers.get();
```

Comment:
this mean that the original value from the URL is ignored.
maybe a better option is to use the broker as the first one, and append the brokers in the list to it?

note that overtiding the value (or preventing double definition, similarly to how we do that in case of user/password)  has some issues:
* it is that its not clear how the URL would look like. e.g. `kafka://` ?
this may not pass as a URL at all
* there migth be a case where user/password are provided via the URL without host. e.g. `kafka://user:password@` which is even uglier.

either way, the behavior should be documented in `notifications.rst`

Reply:
let's me try this.